### ISSUE TITLE
pretrain models 

### ISSUE THREAD

**Issue by robosina** [tinyvision/SOLIDER-PersonSearch#2]
Do you have any pretrained models for person search datasets that you can share?

**Comment by cwhgn** [comment]
You can download the pretrained model from [SOLIDER repo](https://github.com/tinyvision/SOLIDER), and use the convert_model.py to convert it for the person search task.

**Comment by robosina** [comment]
@cwhgn Thank you for your response. I was considering retraining the model for the person search task. However, I currently lack access to a powerful GPU. I was hoping to find a pre-trained model specifically for the person search substream. So, you're suggesting that adapting the base model for this specific task should be enough, correct? and in that case, does retrain will increase the accuracy? I appreciate your help

**Comment by cwhgn** [comment]
SOLIDER model would be stronger than imagenet pre-trained model. However, a model trained on some other person search datasets would be a better pre-trained model for person search tasks, than these base models (SOLIDER/Imagenet). We do not have this kind of models currently, because in my opinion it can not provide a fair comparison due to involving other person search data.
